### ISSUE TITLE
Translate the Images in Etc

### ISSUE THREAD

**Issue by Lazlocke** [Lazlocke/wizardsc#4]
This is a small subset of interface images. In particular, ~250 of them are shop/treasure items. These will be somewhat blocked by the translation of the obj files since they are likely to need to be the same as those translations.

**Comment by Lazlocke** [comment]
Calling this essentially done. Any new problems/unfinished work should have its own issue opened.
